packages/react/src/components/Tree/Tree.tsx

```diff
@@ -39,7 +48,7 @@ export interface TreeProps extends UIComponentProps, ChildrenComponentProps {
 /**
  * Allows users to display data organised in tree-hierarchy.
  */
-class Tree extends UIComponent<ReactProps<TreeProps>> {
+class Tree extends AutoControlledComponent<ReactProps<TreeProps>, any> {
```

Comment:
Let's create an interface for the state:

```tsx
export interface TreeState {
  activeIndex: number[] | number
}
```

```suggestion
class Tree extends AutoControlledComponent<ReactProps<TreeProps>, TreeState> {
```